### PR TITLE
UI: Minor fixes for updating the packages and fixing lint

### PR DESCRIPTION
**Pull request by PuneetPunamiya**
Signed-off-by: Puneet Punamiya <ppunamiy@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/utils/useDebounce.test.tsx [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
   it('it renders useDebounce', (done) => {
-    const { result } = renderHook(() => useDebounce('cli', 400));
-    expect(result.current).toBe('cli');
+    const component = mount(
+      <Provider>
+        <Router>
+          <Resources />
+        </Router>
+      </Provider>
+    );
```
Comment (vinamra28) [inline ui/src/utils/useDebounce.test.tsx:24]: do we need this?

File: ui/src/utils/useDebounce.test.tsx [hunk 2]
```diff
@@ -1,17 +1,46 @@
...
+    const { resources } = root;
+
+    when(
+      () => {
+        return !resources.isLoading;
+      },
+      () => {
+        setTimeout(() => {
+          const resource = resources.filteredResources;
+          expect(resource.length).toBe(7);
```
Comment (vinamra28) [inline ui/src/utils/useDebounce.test.tsx:34]: I guess we can skip this test. WDYT?

File: ui/src/utils/useDebounce.test.tsx [hunk 3]
```diff
@@ -1,17 +1,46 @@
...
+
+          const { result } = renderHook(() => useDebounce('cli', 400));
+          expect(result.current).toBe('cli');
+
+          component.update();
```
Comment (vinamra28) [inline ui/src/utils/useDebounce.test.tsx:39]: this can also be removed I guess. WDYT?

**Review by vinamra28 — COMMENTED** [review]
a couple of comments otherwise looks good

File: ui/src/utils/useDebounce.test.tsx (hunk 1 above)
Comment (PuneetPunamiya) [reply ui/src/utils/useDebounce.test.tsx]: Yes we can remove this

**Review by vinamra28 — COMMENTED** [review]
/lgtm

**Pull request merged**